django_project/utils/views.py

```diff
@@ -4,30 +4,44 @@
 from django.core.cache import cache
 # Create your views here.
 
-def renderHTML(request, subpath=''):
-	soup = getSoup()
-	setMetaTags(soup, subpath)
-	return HttpResponse(soup.prettify())
-
+"""
+	DO NOT MODIFY THESE TWO FUNCTIONS!!
+"""
 #Use this function to set meta tags in soup
 def insertMetaTag(soup, property, content):
```

Comment:
This is a super small nit pick but I think it should be pretty quick to change. In python, the convention is to use snake case for the functions and variable names. Class names use camel case. Could you please change the function names to snake case?